devel/zeal/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           github 1.0
+PortGroup           qt5 1.0
+
+github.setup        zealdocs zeal 0.6.1 v
+categories          devel
+platforms           darwin
+license             GPL-3+
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         Offline documentation browser
+long_description    Zeal is a simple offline documentation \
+                    browser inspired by Dash.
+homepage            https://zealdocs.org/
+
+checksums           rmd160  4149ec839f0bfd3a73bb452fbc08152145d73f9d \
+                    sha256  084fb42bcc3756c431cefb8ceb65b815625b178a224bb24c7e605f05340ff10d \
+                    size    1064016
+
+depends_lib-append  port:libarchive \
+                    port:qt5-qtwebkit \
```

Comment:
I *think* this could/should be specified as `qt5.depends_component qtwebkit` instead; right now you'll always request the latest version `qt5-qtwebkit` whereas a system might be able to only install `qt511-qtwebkit`. The `qt5` PG should take care of these situations

Reply:
Yes, that is my understanding of how the qt5 portgroup works.

Reply:
Fixed